lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.scss

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-vulnerability-indicator {
+  align-items: center;
+  box-sizing: content-box;
+  border: 1px solid;
+  border-left-width: 8px;
+  border-radius: $nx-border-radius;
+  display: inline-flex;
+  font-size: $nx-font-size-s;
+  height: 30px;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: max-content;
+}
+
+.nx-vulnerability-indicator__count {
+  @include bold();
+}
+
+.nx-vulnerability-indicator__text {
+  margin-left: $nx-spacing-xs;
+  text-transform: uppercase;
+}
+
+.nx-vulnerability-indicator--critical {
+  border-color: $nx-vulnerability-indicator-critical;
+}
+
+.nx-vulnerability-indicator--severe {
+  border-color: $nx-vulnerability-indicator-severe;
+}
+
+.nx-vulnerability-indicator--moderate {
+  border-color: $nx-vulnerability-indicator-moderate;
+}
+
+.nx-vulnerability-indicator--low {
+  border-color: $nx-vulnerability-indicator-low;
+}
+
+.nx-vulnerability-indicator-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-vulnerability-indicator-container--row {
+  @include container-horizontal;
+
+  .nx-vulnerability-indicator {
+    margin-right: $nx-spacing-md;
```

Comment:
If the containers all used `grid` we could use `gap` which would be simpler than messing with these margins

Reply:
> If the containers all used `grid` we could use `gap` which would be simpler than messing with these margins

Yep. https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/0c49ff6f35705c7d71d2c31f9fccde3215fe17b7 & https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/e876a57eacad29d4171059706217dd98cfec0631

The 2x2 layout differs slightly from the mockup but I have conferred with @dturner32 and he is cool with the change

Reply:
> The 2x2 layout differs slightly from the mockup but I have conferred with @dturner32 and he is cool with the change

I guess that does point out the question of how things are supposed to adjust as some numbers get bigger but not others.  Personally I think it looks better in the mockup, and I'd be willing to put up with adding extra DOM elements to get it to match that.

Reply:
> > The 2x2 layout differs slightly from the mockup but I have conferred with @dturner32 and he is cool with the change
> 
> I guess that does point out the question of how things are supposed to adjust as some numbers get bigger but not others. Personally I think it looks better in the mockup, and I'd be willing to put up with adding extra DOM elements to get it to match that.

It turns out that removing `max-width: content` as suggested below lead to a layout that I think looks better all round. The row counters fit to their natural widths and the column layouts expand to the widest counter in the column. As a result the column & grid layouts have clean right edges and a nice square grid. The cost is some extra white-space on the right interior of the counters. I showed this to @dturner32 and he likes it and will be changing the mockups: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/1c0d8a8da7f415a5bc409b6a2f5f9317ce5bd3a7